### PR TITLE
Create rito.json

### PR DESCRIPTION
**Pull request by bartekprtc**

### CODE REVIEW

**Review by stelar7 — APPROVED** [review]

**Review by stelar7 — CHANGES_REQUESTED** [review]
Workflow fails, possibly due to the trailing ,

**Comment by bartekprtc** [comment]
Done, my bad

**Comment by AoshiW** [comment]
you should also add some tags

File: libraries/cpp/rito.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "owner": "bartekprtc",
+    "repo": "rito",
+    "description": "Rito is a simple, crossplatform (Windows and Linux) C++20 library interfacing with Riot services (i.e. Riot REST API and League of Legends client).",
+    "language": "cpp",
+    "links": [
+        {
+            "name": "Source",
+            "url": "https://github.com/bartekprtc/rito"
```
Comment (BlossomiShymae) [inline libraries/cpp/rito.json:9]: Botty adds GitHub source links automatically. This can be removed. :>

Comment (bartekprtc) [reply libraries/cpp/rito.json]: removed, thanks!

**Review by stelar7 — APPROVED** [review]

**Pull request merged**